certified-connectors/Haiilo/apiDefinition.swagger.json

```diff
@@ -4,8 +4,8 @@
   "definitions": {},
```

Comment:
@thanh-nguyen The 'x-ms-connector-metadata' property is required. Please see out template [here](https://github.com/microsoft/PowerPlatformConnectors/blob/dev/templates/certified-connectors/apiDefinition.swagger.json).
```
  "x-ms-connector-metadata": [
    {
      "propertyName": "Website",
      "propertyValue": ""
    },
    {
      "propertyName": "Privacy policy",
      "propertyValue": ""
    },
    {
      "propertyName": "Categories",
      "propertyValue": ""
    }
  ]
```